ui/admin/tests/e2e/tests/credential-store-static.spec.js

```diff
@@ -146,3 +146,211 @@ test('Static Credential Store (User & Key Pair)', async ({ page }) => {
+  await page.getByRole('link', { name: 'Credentials', exact: true }).click();
+  await page.getByRole('link', { name: 'New', exact: true }).click();
+  await page.getByLabel('Name', { exact: true }).fill(credentialName);
+  await page.getByLabel('Description').fill('This is an automated test');
+  await page.getByRole('group', { name: 'Type' }).getByLabel('JSON').click();
+  await page.getByText('{}').click();
+  const testName = 'name-json';
+  const testPassword = 'password-json';
+  await page.keyboard.type(
+    `"username": "${testName}", "password": "${testPassword}"`
+  );
+  await page.getByRole('button', { name: 'Save' }).click();
+  await expect(
+    page.getByRole('alert').getByText('Success', { exact: true })
+  ).toBeVisible();
+  await page.getByRole('button', { name: 'Dismiss' }).click();
+  await expect(page.getByRole('link', { name: credentialName })).toBeVisible();
+
+  await page
+    .getByRole('navigation', { name: 'Resources' })
+    .getByRole('link', { name: 'Targets' })
+    .click();
+  await page.getByRole('link', { name: targetName }).click();
+  await page
+    .getByRole('link', { name: 'Brokered Credentials', exact: true })
+    .click();
+  await page
+    .getByRole('article')
+    .getByRole('link', { name: 'Add Brokered Credentials', exact: true })
+    .click();
+  await page
+    .getByRole('cell', { name: credentialName })
+    .locator('..')
+    .getByRole('checkbox')
+    .click({ force: true });
+  await page
+    .getByRole('button', { name: 'Add Brokered Credentials', exact: true })
+    .click();
+  await expect(
+    page.getByRole('alert').getByText('Success', { exact: true })
+  ).toBeVisible();
+  await page.getByRole('button', { name: 'Dismiss' }).click();
+  await expect(page.getByRole('link', { name: credentialName })).toBeVisible();
+
+  await authenticateBoundaryCli();
+  const orgs = JSON.parse(execSync('boundary scopes list -format json'));
+  const org = orgs.items.filter((obj) => obj.name == orgName)[0];
+  const projects = JSON.parse(
+    execSync(`boundary scopes list -format json -scope-id ${org.id}`)
+  );
+  const project = projects.items.filter((obj) => obj.name == projectName)[0];
+  const targets = JSON.parse(
+    execSync(`boundary targets list -format json -scope-id ${project.id}`)
+  );
+  const target = targets.items.filter((obj) => obj.name == targetName)[0];
+
+  const session = JSON.parse(
+    execSync(`boundary targets authorize-session -id ${target.id} -format json`)
+  );
+  const retrievedUser = session.item.credentials[0].credential.username;
+  const retrievedPassword = session.item.credentials[0].credential.password;
+
+  if (testName != retrievedUser) {
+    throw new Error(
+      'Stored User does not match. EXPECTED: ' +
+        process.env.E2E_SSH_USER +
+        ', ACTUAL: ' +
+        retrievedUser
+    );
+  }
+  if (testPassword != retrievedPassword) {
+    throw new Error('Stored Password does not match');
+  }
```

Comment:
We might be able to use the more explicit error message here since this password is just a test one.